spec/spec_helper_acceptance.rb

```diff
@@ -40,15 +41,13 @@ def install_modules_on(host)
   options['is_puppetserver'] = true
   master['puppetservice'] = 'puppetserver'
   master['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
-  master['type'] = 'aio'
-  install_puppet_agent_on master, {}
-  install_package master, 'puppetserver'
-  master['use-service'] = true
+  master['type'] = 'foss'
```

Comment:
Is breaking if we use `aio`?
https://github.com/puppetlabs/beaker/blob/master/docs/concepts/types_puppet_4_and_the_all_in_one_agent.md